### PR TITLE
Proof RESTARTS_ROUTINE (#251)

### PR DESCRIPTION
**Pull request by keller0**
"ERRASIBLE"  appeared in WAITLIST.agc too, So I think it's not a typo.
p1

### CODE REVIEW

File: Comanche055/RESTARTS_ROUTINE.agc [hunk 1]
```diff
@@ -162,13 +162,13 @@ ITSLIKEB	CA	RTRNCADR	# TYPE B, SO STORE RETURN IN
 
 ITSAJOB		NDX	TEMP2G		# NOW ADD THE PRIORITY AND LET'S GO
```
Comment (wopian) [inline Comanche055/RESTARTS_ROUTINE.agc]: ✨ Change `LET'S` to `LET:S`
✨ Also lines `0091` and `0108` that need `'` changed to `:` (see [p1416](http://www.ibiblio.org/apollo/ScansForConversion/Comanche055/1416.jpg) scans)

File: Comanche055/RESTARTS_ROUTINE.agc [hunk 2]
```diff
@@ -210,7 +210,7 @@ CONTBL2		EXTEND			# FIND OUT WHAT'S IN THE TABLE
 		MASK	BIT10		# THIS SHOULD BE ONE IF WE HAVE -BB
 		CCS	A		# FOR THAT MATTER SO SHOULD BE BITS 9,8,7,
 					# 6,5, AND LAST BUT NOT LEAST (PERHAPS NOT
-					# IN IMPORTANCE ANYWAY.  BUT 4
+					# IN IMPORTANCE ANYWAY.  BIT 4
```
Comment (wopian) [inline Comanche055/RESTARTS_ROUTINE.agc]: ⚠️ Single space before `BIT` (see [`A0133`](http://www.ibiblio.org/apollo/ScansForConversion/Comanche055/1417.jpg))

File: Comanche055/RESTARTS_ROUTINE.agc [hunk 3]
```diff
@@ -237,7 +237,8 @@ ITSLGCL1	LXCH	GOLOC +1	# OBTAIN THE CORRECT E BANK
 		LXCH	GOLOC +1	# RESTORE OUR E AND F BANK
 		LXCH	BB		# RESTORE THE TASKS E AND F BANKS
 		LXCH	GOLOC +1	# AND PRESERVE OUR L
-		TCF	ITSLGCL2	# NOT GET'S PROCESS THIS LONGCALL
+
+		TCF	ITSLGCL2	# NOW LET'S PROCESS THIS LONGCALL
```
Comment (wopian) [inline Comanche055/RESTARTS_ROUTINE.agc]: ✨ Change `LET'S` to `LET:S`

File: Comanche055/RESTARTS_ROUTINE.agc [hunk 4]
```diff
@@ -305,13 +307,13 @@ ITSEVEN		CA	TEMPSWCH	# SET FOR EITHER THE SECOND PART OF THE
...
+		TCF	CONTBL2		# NOW PROCESS WHAT IS IN THE TABLE
 
 PHSPART2	CA	THREE		# SET THE POINTER FOR THE SECOND HALF OF
 		ADS	POINTER		# THE TABLE
 
 		CA	RTRNCADR	# THIS WILL BE OUR LAST TIME THROUGH THE
-		TS	GOLOC +2	# EVEN TABLE, SO AFTER IT GET THE NEXT
+		TS	GOLOC +2	# EVEN TABLE , SO AFTER IT  GET THE NEXT
 					# GROUP
 		TCF	CONTBL2		# SO LET'S GET THE SECOND ENTRY IN THE TBL
```
Comment (wopian) [inline Comanche055/RESTARTS_ROUTINE.agc]: ✨ Change `LET'S` to `LET:S`

**Comment by keller0** [comment]
All these `LET'S` to `LET:S` has ben changed.
But how about this one?
h

**Comment by wopian** [comment]
@keller0 `LET-S` 👍

**Comment by keller0** [comment]
Sorry, After reading more code, I realized that `ERRASIBLE` is really a typo. I will change those in #341 .

**Pull request merged**